plugins/SegmentEditor/javascripts/Segmentation.js

```diff
@@ -80,14 +80,14 @@ Segmentation = (function($) {
                 var currentDecoded = piwikHelper.htmlDecode(current);
                 var selector = 'div.segmentList ul li[data-definition="'+currentDecoded+'"]';
                 var foundItems = $(selector, this.target);
-                var title = $('<strong></strong>');
+
                 if( foundItems.length > 0) {
-                    var name = $(foundItems).first().find("span.segname").text();
-                    title.text(name);
+                    var idSegment = $(foundItems).first().attr('data-idsegment');
+                    var title = getSegmentName( getSegmentFromId(idSegment));
                 } else {
-                    title.text(_pk_translate('SegmentEditor_CustomSegment'));
+                    title = _pk_translate('SegmentEditor_CustomSegment');
                 }
-                segmentationTitle.html(title);
+                segmentationTitle.html( "<strong>" + title + "</strong>");
```

Comment:
Wouldn't it be better to remove the `<strong>` here and to add a css class (having `font-weight: bold`) instead. That would make it easier for custom designs to overwrite the css at that point.

Note: The `.text()` below does not remove a `<strong>` (if set before), it simply replaces the text within. Not sure if that is expected behavior.